ai/Util/role_mapper.py

```diff
@@ -0,0 +1,34 @@
+from .role import Role
+
+class RoleMapper(object):
+    LOCKED_ROLES = [Role.GOALKEEPER]
+
+    def __init__(self):
+        self.roles_translation = {r: None for r in Role}
+
+    def map_by_player(self, desired_map):
+        saved_roles = self._save_locked_roles(self.roles_translation)
+        base_map = self._remove_undesired_roles(self.roles_translation, desired_map)
+
+        for key, value in desired_map.items():
+            base_map[key] = value
+
+        results = self._apply_locked_roles(saved_roles, base_map)
+        # if len(results.values()) > len(set(results.values())):
```

Comment:
Will need to reimplent this check in some way, so that we can be sure that one robot is not mapped to two role at once